modules/python/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ add_subdirectory(python3)
 
 else()  # standalone build
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION ${MIN_VER_CMAKE})
```

Comment:
> standalone

`MIN_VER_CMAKE` may be not available for standalone bindings, samples, hal samples.

Reply:
Lets bump to 2.8.12 for other cases to avoid warning.

Reply:
I took this "other cases" as standalone bindings, samples and hal samples.
I also bumped ones in the "toolchain.cmake" in platform directory, please let me know if that's along with your expectation.

Only remaining `cmake_minimum_required` that I didn't touch is the one in the doc directory